bluemira/geometry/optimisation.py

```diff
@@ -46,44 +121,54 @@ class GeometryOptimisationProblem(abc.ABC):
         Optimiser instance to use in the optimisation problem
     """
 
-    def __init__(self, parameterisation: GeometryParameterisation, optimiser: Optimiser):
-        self.parameterisation = parameterisation
-        self.optimiser = optimiser
-        self.optimiser.build_optimiser(parameterisation.variables.n_free_variables)
-        self.optimiser.set_lower_bounds(np.zeros(optimiser.n_variables))
-        self.optimiser.set_upper_bounds(np.ones(optimiser.n_variables))
-        self.optimiser.set_objective_function(self.f_objective)
+    def __init__(
+        self,
+        parameterisation: GeometryParameterisation,
```

Comment:
Possibly worth calling this input `geometry_parameterisation` to make it a little clearer to a reader the difference in usage between this and _parameterisation (although I know that they will reference the same object after the super().__init__ call).